.github/workflows/submodule-pin-check.yml

```diff
@@ -1,21 +1,48 @@
-name: Submodule Pin Check
+name: Merge Checks
 
 on:
   pull_request:
```

Comment:
As-is this won't work for 3rd party PRs as they're missing permissions. We should change this to `pull_request_target` as in merge-checks.yml

Reply:
Also note that it won't show up until this PR is merged with pull_request_target, due to how it works

Reply:
Done.